ament_package/package.py

```diff
@@ -98,9 +98,21 @@ def validate(self):
 
         if not self.name:
             errors.append('Package name must not be empty')
-        # only allow lower case alphanummeric characters and underscores
+        # assume packages are catkin unless a build_type is exported.
+        build_type = 'catkin'
+        build_types = [ex.contents for ex in self.exports if ex.tagname == 'build_type']
+        if len(build_types) > 1:
+            # this is bad, but first I must ask if we should make this an error.
```

Comment:
[catkin_pkg raises InvalidPackage](https://github.com/ros-infrastructure/catkin_pkg/blob/0ec1d3d30164aed656cbb4e823bbfffa836baae8/src/catkin_pkg/package.py#L153) if multiple build_types are present. I think it makes sense to add an error here in order to do likewise but I wanted to check before doing so.